src/QueryStrategies/BatchQueryStrategies/ClusterBatchQs.jl

```diff
@@ -0,0 +1,40 @@
+struct ClusterBatchQs <: BatchPQs
+    model::SVDD.OCClassifier
+    sequentialQs::SequentialPQs
+    k::Int
+    max_iterations::Int
+
+    function ClusterBatchQs(model::SVDD.OCClassifier, sequential_strategy::SequentialPQs; k::Int = 0, max_iterations::Int = 1000)::ClusterBatchQs
+        (k < 1) && throw(ArgumentError("Invalid batch size k=$(k)."))
```

Comment:
This signature and the check for k<1 is basically a requirement to set k.

See comment on AllRandomBatchQs